webpack/index.ejs

```diff
@@ -9,7 +9,7 @@
       <title>AppBuilder</title>
       <!-- https://www.filamentgroup.com/lab/load-css-simpler/#the-code -->
       <link href="/__getCookie" rel="stylesheet" media="print" onload="this.media='all'" />
-      <script type='text/javascript' src='/assets/dependencies/sails.io.js' reconnection="true" defer></script>
+      <script type='text/javascript' src='/config/preloader'></script>
```

Comment:
Can you add `defer` here? This will allow the browser to render our preload UI and keep our first contentful paint score high.

Reply:
ok, made some changes to enable this.  

turns out, this is a good update and shaves some more time off the load.